terraform/aws/analytical-platform-data-production/s3-glue-crawler/glue-crawler.tf

```
@@ -0,0 +1,9 @@
+resource "aws_glue_crawler" "vmcs_data_crawler" {
+  database_name = aws_glue_catalog_database.alpha_vcms_data_glue_catalog.name
+  name          = "vcms-data-crawler"
```

update name and resource name to be use alpha prefix, drop `crawler` from resource name
```suggestion
resource "aws_glue_crawler" "alpha_vmcs_data" {
  database_name = aws_glue_catalog_database.alpha_vcms_data_glue_catalog.name
  name          = "alpha-vcms-data"
```